content/apps/databases.md

```diff
@@ -63,10 +67,25 @@ To access a service database, use the [cf-ssh]({{< relref "getting-started/one-o
 
 You should now have an open `psql` terminal connected to the service database.
 
+### *GovCloud environment:* Using cf ssh
+
+To access a service database, use the [cf ssh]({{< relref "getting-started/one-off-tasks.md#cf-ssh" >}}) CLI command to login to an instance that is bound to the service and download the `psql` binary to that instance:
+
+    cf ssh APP_NAME
+    curl https://s3.amazonaws.com/18f-cf-cli/psql-9.4.4-ubuntu-14.04.tar.gz | tar xvz
+    ./psql/bin/psql $DATABASE_URL
```

Comment:
It's a bigger topic... Even if they tunnel they have to have the client *somewhere*. Given most agency laptops/desktops are locked down pretty tight, so they might actually have to stick with the client-installed-in-container route. There's also the potential for [using scp](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#other-ssh-access) to actually get files dump files in and out. For future!